File: src/main/java/redcrafter07/processed/block/tile_entities/capabilities/SimpleInputItemStore.java
package redcrafter07.processed.block.tile_entities.capabilities;

import net.minecraft.core.NonNullList;
import net.minecraft.world.item.ItemStack;

public class SimpleInputItemStore extends
        ProcessedItemStackHandler {
    public SimpleInputItemStore(NonNullList<ItemStack> items) {
        super(items);
    }

    public SimpleInputItemStore(int size) {
        super(size);
    }

    public SimpleInputItemStore() {
        super();
    }
}

File: src/main/kotlin/redcrafter07/processed/block/tile_entities/capabilities/SimpleInputItemStore.kt
package redcrafter07.processed.block.tile_entities.capabilities

import net.minecraft.core.NonNullList
import net.minecraft.world.item.ItemStack

open class SimpleInputItemStore(items: NonNullList<ItemStack>) :
    ProcessedItemStackHandler(items) {
    constructor(size: Int) : this(NonNullList.withSize(size, ItemStack.EMPTY))
    constructor() : this(1)
}